generator/_templates/beat/{beat}/main.go

```diff
@@ -5,6 +5,7 @@ import (
 
 	"{beat_path}/cmd"
 
+	// Make sure all your modules and metricsets are linked in this file
```

Comment:
This one doesn't generate any warning, even after running the generator. I still added it because I figured it would be clearer to someone using that generator.
The content is the same as in the metricbeat generator: https://github.com/elastic/beats/blob/master/generator/_templates/metricbeat/%7Bbeat%7D/main.go#L8